api/config/config.example.json

```
@@ -15,5 +15,11 @@
   "ledSignIp": "192.168.4.1",
   "GENERAL_API_URL": "http://localhost:8080/",
   "LOGGING_API_URL": "http://localhost:8081/",
-  "MAILER_API_URL": "http://localhost:8082/"
+  "MAILER_API_URL": "http://localhost:8082/",
+  "discordApiKeys": {
+    "CLIENT_SECRET": "XXXXXXXXXXXXXXXXXX",
+    "CLIENT_ID": "XXXXXXXXXXXXXXXXXX",
+    "REDIRECT_URI": "http%3A%2F%2Flocalhost%3A8080%2Fapi%2Fuser%2Fcallback&response_type=code&scope=identify",
```

consider making 2 fields like

```suggestion
    "REDIRECT_URI_DEV": "http%3A%2F%2Flocalhost%3A8080%2Fapi%2Fuser%2Fcallback&response_type=code&scope=identify",
    "REDIRECT_URI_PROD": "http%3A%2F%2Fsce.sjsu.edu%2Fapi%2Fuser%2Fcallback&response_type=code&scope=identify",
```

then you can conditionally redirect to either dev or prod if `process.env.NODE_ENV === 'production'`

---

alright, will update the discord dev-app info with the prod link when the website comes back. side note - where would that line of code `process.env.NODE_ENV === 'production'` go?

---

if you change the axios.post back to use [this](https://github.com/SCE-Development/Core-v4/pull/562#discussion_r464208028)

you can do

```js
const runningInProduciton = process.env.NODE_ENV === 'production';
// stuff ...
// in axios.post
{
  redirect_uri: runningInProduciton ? REDIRECT_URI_PROD  : REDIRECT_URI_DEV
}
```